Hestia/Hestia/frontend/DevicesScreen/TableSourceDevicesMain.cs

```diff
@@ -96,37 +102,41 @@ public override UITableViewCell GetCell(UITableView tableView, NSIndexPath index
                 //    {
                 //        cell.ImageView.Image = UIImage.FromBundle("Images/lock.png");
```

Comment:
commented out

Reply:
Not sure if we are still going to use it, for now removed in 66c8a1fb4f8e4adba71d6222e28fe493127537af